gef.py

```diff
@@ -6286,14 +6287,58 @@ def do_invoke(self, _: List[str], **kwargs: Any) -> None:
             gef_print(current_chunk.psprint())
         return
 
+class GlibcHeapChunkSummary:
+    def __init__(self):
+        self.count = 0
+        self.total_size = 0
+
+    def process_chunk(self, chunk: GlibcChunk) -> None:
+        self.count += 1
+        self.total_size += chunk.size
+
+class GlibcHeapArenaSummary:
+    def __init__(self) -> None:
+        self.size_distribution = {}
+        self.flag_distribution = {
+            "PREV_INUSE": GlibcHeapChunkSummary(),
+            "IS_MMAPPED": GlibcHeapChunkSummary(),
+            "NON_MAIN_ARENA": GlibcHeapChunkSummary()
+        }
+
+    def process_chunk(self, chunk: GlibcChunk) -> None:
+        per_size_summary = self.size_distribution.get(chunk.size, None)
+        if per_size_summary is None:
+            per_size_summary = GlibcHeapChunkSummary()
+            self.size_distribution[chunk.size] = per_size_summary
+        per_size_summary.process_chunk(chunk)
+
+        if chunk.has_p_bit():
+            self.flag_distribution["PREV_INUSE"].process_chunk(chunk)
+        if chunk.has_m_bit():
+            self.flag_distribution["IS_MAPPED"].process_chunk(chunk)
+        if chunk.has_n_bit():
+            self.flag_distribution["NON_MAIN_ARENA"].process_chunk(chunk)
+
+    def print(self) -> None:
+        gef_print(f"== Chunk distribution by size ==")
+        gef_print("{:<10s}\t{:<10s}\t{:s}".format("ChunkSize", "Count", "TotalSize"))
```

Comment:
should add units e.g. `bytes` or `B`

Reply:
I changed the title to ChunkBytes and TotalBytes, which should be more clear.